scripts/deploy.sh

```
@@ -0,0 +1,107 @@
+
+# Backup current version
+if [ -d "$CURRENT_DIR" ] && [ "$(ls -A $CURRENT_DIR)" ]; then
+    echo "💾 Creating backup..."
+    BACKUP_NAME="backup-$(date +%Y%m%d-%H%M%S)"
+    cp -r $CURRENT_DIR $BACKUP_DIR/$BACKUP_NAME
+    echo "✅ Backup created: $BACKUP_NAME"
+fi
+
+# Clean current directory
+echo "🧹 Cleaning current directory..."
+rm -rf $CURRENT_DIR/*
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to /* .

(SC2115)